lib/win32/certstore/mixin/unicode.rb

```diff
@@ -0,0 +1,49 @@
+#
+# Author:: Vasundhara Jagdale (<vasundhara.jagdale@msystechnologies.com>)
+# Copyright:: Copyright (c) 2017 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'helpers'
+
+module Win32::Certstore::Mixin::Unicode
+end 
```

Comment:
There's nothing defined inside. Do we need this?

Reply:
I think since `helper` and `unicode` file both contain string functions, we can think of some way to merge these.

Reply:
I have created unicode.rb to copy the same behavior as it is in chef where they have monkey patch for FFI::Pointer in which they have added read_wstring method https://github.com/chef/chef/blob/5b9b829d283afceaf36ba060d42d51729010b2e9/lib/chef/win32/unicode.rb and kept the other methods in helper.rb as those are helper methods for wide string and for shellout command which are also copied from chef https://github.com/chef/chef/blob/master/lib/chef/mixin/wide_string.rb

Reply:
yes we require do define empty module since I need to require this module in other files. As I mentioned I just duplicated what there in chef unicode.rb

Reply:
I agree with Nimisha, probably if the helper module was named wide_string or something that is not generic "helper" that might sort it out a bit.